test/integration/testLandUtils.js

```diff
@@ -0,0 +1,51 @@
+const LandUtils = artifacts.require('./utils/LandUtils.sol');
```

Comment:
This is not an integration test, just a unit. We already have one for LandUtils in the corresponding folder